spec/integration/rules_spec.rb

```diff
@@ -11,6 +11,7 @@
       rule_count: 1,
       account: @account
     )
+    @profile.rules.update(precedence: 1)
```

Comment:
Why every rule has the same precedence?

Reply:
It doesn't really matter in the openapi spec :see_no_evil: I did not want to write 5 lines :grin: